std/range/package.d

```diff
@@ -8609,10 +8609,18 @@ unittest
     if it were passed to it, the original range is $(I not) copied but is
     consumed as if it were a reference type.
 
-    Note that $(D save) works as normal and operates on a new range, so if
-    $(D save) is ever called on the RefRange, then no operations on the saved
-    range will affect the original.
-
+    Note:
+        $(D save) works as normal and operates on a new range, so if
+        $(D save) is ever called on the `RefRange`, then no operations on the
+        saved range will affect the original.
+
+    Params:
+        range = the range to construct the $(LREF RefRange) off of
```

Comment:
Ditto `_range`.

The LREF generates broken HTML because "RefRange" is being auto-highlighted. Can fix that with `_RefRange`, but since we're already in the documentation for RefRange, I suggest to just drop the LREF macro. Same below.